client/landing/stepper/declarative-flow/internals/steps-repository/sensei-setup/sensei-step-error/components.ts

```diff
@@ -0,0 +1,38 @@
+import { ErrorIcon as UnstyledErrorIcon } from '@automattic/composite-checkout/src/components/shared-icons';
+import styled from '@emotion/styled';
+
+export const Container = styled.div`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	flex: 1;
+`;
+
+export const ErrorIcon = styled( UnstyledErrorIcon )`
+	width: 40px;
+	height: 40px;
+	path {
+		fill: var( --studio-orange-40 );
+	}
+`;
+
+export const Title = styled.h1`
+	font-family: 'Recoleta';
```

Comment:
Recommend including fallback fonts after "Recoleta" in case it doesn't load for some reason, see https://github.com/Automattic/wp-calypso/blob/b74abfa38d482735dfde4270f5312a732c58b783/packages/components/src/styles/_typography.scss#L1-L3

Reply:
Added fallback fonts in 1783b4ca14565edeb9765acb74c44e250e515a8a